Source/WebCore/animation/CSSPropertyAnimation.h

```diff
@@ -50,7 +50,7 @@ class CSSPropertyAnimation {
     static int getNumProperties();
 
     static void blendProperties(const CSSPropertyBlendingClient*, CSSPropertyID, RenderStyle& destination, const RenderStyle& from, const RenderStyle& to, double progress, CompositeOperation, IterationCompositeOperation = IterationCompositeOperation::Replace, double currentIteration = 0);
-    static void blendCustomProperty(const AtomString&, RenderStyle& destination, const RenderStyle& from, const RenderStyle& to, double progress);
+    static void blendCustomProperty(const CSSPropertyBlendingClient*, const AtomString&, RenderStyle& destination, const RenderStyle& from, const RenderStyle& to, double progress, CompositeOperation, IterationCompositeOperation = IterationCompositeOperation::Replace, double currentIteration = 0);
```

Comment:
Can CSSPropertyBlendingClient be a reference? All callers are providing it.

Reply:
Good point. I'll fix this for the new call methods and do a refactor for all the other places in https://bugs.webkit.org/show_bug.cgi?id=249320.